test/js/web/console/console-log.js

```diff
@@ -14,6 +14,7 @@ console.log(Infinity);
 console.log(-Infinity);
 console.log(Symbol("Symbol Description"));
 console.log(new Date(Math.pow(2, 34) * 56));
+console.log(new String("Hello"));
```

Comment:
```suggestion
console.log(new String("Hello"));
console.log(new Number(5));
console.log(new Boolean(true));
```

While we're here, can you add test coverage for these other varieties of boxed primitives? Bun does already handle them the same way Node does (unlike strings) but it'd be nice to make sure we remain consistent.

Reply:
Done.